estate/models/estate_property.py

```diff
@@ -0,0 +1,79 @@
+from odoo import fields, models, api
+from odoo.tools import date_utils as du
+from datetime import date
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "A property module that adds the property as a listing"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(copy=False, default=du.add(date.today(), months=3))
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(readonly=True, copy=False)
+    bedrooms = fields.Integer(default=2)
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection(
+        string="Orientation",
+        selection=[
+            ("north", "North"),
+            ("south", "South"),
+            ("east", "East"),
+            ("west", "West"),
+        ],
+    )
+    total_area = fields.Float(compute="_compute_area")
+    active = fields.Boolean(default=True)
+    state = fields.Selection(
+        string="State",
+        selection=[
+            ("new", "New"),
+            ("offer-received", "Offer Received"),
+            ("offer-accepted", "Offer Accepted"),
+            ("sold", "Sold"),
+            ("cancelled", "Cancelled"),
+        ],
+        required=True,
+        copy=False,
+        default="new",
+    )
+    property_type_id = fields.Many2one("estate.property.type")
+    buyer_id = fields.Many2one("res.partner", copy=False)
+    seller_id = fields.Many2one(
+        "res.users", name="Salesperson", default=lambda self: self.env.user
+    )
+    tag_ids = fields.Many2many("estate.property.tag", string="Tags")
+    offers_ids = fields.One2many(
+        "estate.property.offer", "property_id", string="Offers"
+    )
+    best_price = fields.Float(
+        compute="_compute_best_price", readonly=True, string="Best Offer"
+    )
+
+    @api.depends("living_area", "garden", "garden_area")
+    def _compute_area(self):
```

Comment:
nitpick: for computes, you can just name them `_compute_<fieldname>` to keep consistency 
```suggestion
    def _compute_total_area(self):
```